aspnetcore/tutorials/signalr-typescript-webpack.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Use ASP.NET Core SignalR with TypeScript and Webpack
+title: "Tutorial: Get started with ASP.NET Core SignalR using TypeScript and Webpack"
```

Comment:
```suggestion
title: Tutorial: Get started with ASP.NET Core SignalR using TypeScript and Webpack
```

Reply:
Why the quotes?

Reply:
> The `:` in the value makes them required, otherwise it's invalid YAML.

Most of our topics don't use `""`

Reply:
@Rick-Anderson, maybe you are thinking of the H1.  The meta title could not have a `:`, it would treat it as another meta entry.

Reply:
Do a search, most don't have `"`
<img width="380" alt="image" src="https://user-images.githubusercontent.com/3605364/157322444-0343aa32-980b-400d-b673-c293c105fba5.png">

Reply:
@guardrex  can you explain the need for quotes here?

Reply:
Never mind, I get it, there's an extra `:`, that's why we don't normally have quotes.